.github/workflows/Build.yml

```diff
@@ -13,26 +13,63 @@ jobs:
       matrix:
         include:
           - os: ubuntu-16.04
+            NAME_ADD: static
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=0
           - os: ubuntu-18.04
+            NAME_ADD: static
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=0
           - os: ubuntu-20.04
+            NAME_ADD: static
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=0
+          - os: ubuntu-20.04
+            NAME_ADD: shared
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=1
+            OUTPUT_LIB: libvvdec.so
+          - os: macos-10.15
+            NAME_ADD: static
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=0
           - os: macos-10.15
+            NAME_ADD: shared
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=1
+            OUTPUT_LIB: libvvdec.dylib
+          - os: windows-2019
+            NAME_ADD: shared
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=1
+            OUTPUT_LIB: vvdec.dll
           - os: windows-2019
+            NAME_ADD: static
+            CMAKE_ADD: -DBUILD_SHARED_LIBS=0
     steps:
     - uses: actions/checkout@v2
     - run: git fetch --prune --unshallow
     - name: Build Linux / mac
       run: |
         mkdir build
         cd build
-        cmake .. -DCMAKE_BUILD_TYPE=Release
+        cmake ${{matrix.CMAKE_ADD}} -DCMAKE_BUILD_TYPE=Release ..
         make -j$(nproc)
       if: matrix.os != 'windows-2019'
     - name: Build Windows
       run: |
         mkdir build
         cd build
         call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
-        cmake .. -DCMAKE_BUILD_TYPE=Release
-        msbuild vvdec.sln
+        cmake ${{matrix.CMAKE_ADD}} ..
+        msbuild /property:Configuration=Release vvdec.sln
```

Comment:
This was actually a bug. This is how you build the release version on windows. Before, the debug version was built.

Reply:
Perhaps it would be better to use `cmake --build . --config Release` so you don't have to use msbuild directly?

Reply:
Very good idea. Then you could even switch to a different build system like Ninja ... But for a project this small that is not really necessary.